docker-compose.yml

```diff
@@ -23,6 +23,7 @@ services:
       dockerfile: ./docker/images/dev/Dockerfile
     image: marsha:dev
     env_file: env.d/development
+    env_file: env.d/development_secret
```

Comment:
@mbenadda you can now put your AWS secrets for development in "env.d/development_secret".
See env.d/test to see what the expected secrets are:
```
# AWS User
DJANGO_AWS_ACCESS_KEY_ID=aws-access-key-id
DJANGO_AWS_SECRET_ACCESS_KEY=aws-secret-access-key

# CloudFront User (has to be a root user, cannot be an IAM user)
DJANGO_CLOUDFRONT_ACCESS_KEY_ID=cloudfront-access-key-id
DJANGO_CLOUDFRONT_URL=https://abc.cloudfront.net
```